src/olympia/reviewers/urls.py

```diff
@@ -73,6 +73,11 @@ def queue_urls():
         views.whiteboard,
         name='reviewers.whiteboard',
     ),
+    re_path(
```

Comment:
Nit: maybe calling the url something like

`decision/<decision_id>`

Reply:
Like, instead of held_action_review? Or in addition to it?

Reply:
instead of.. this page is for showing a decision right? held, action, and review have nothing to do with cinder or decision so it feels like decision is the most descriptive word.. but just an singular opiniomation.

Reply:
The thinking was it was a review for the action associated with the decision, which was held from being carried out pending that review.  I _do_ think we need "review" in the url, but "/decision_review"? 🤷